extensions/ruby/package.json

```diff
@@ -12,12 +12,12 @@
 	"contributes": {
 		"languages": [{
 			"id": "ruby",
-			"extensions": [ ".rb", ".rbx", ".rjs", ".gemspec", ".rake", ".ru", ".erb" ],
+			"extensions": [ ".rb", ".rbx", ".rjs", ".gemspec", ".rake", ".ru", ".erb",".podspec" ],
 			"filenames": [ "rakefile", "gemfile", "guardfile", "podfile", "capfile" ],
 			"aliases": [ "Ruby", "rb" ],
 			"firstLine": "^#!\\s*/.*\\bruby\\b",
 			"configuration": "./language-configuration.json"
-		}],
+    }],
```

Comment:
nit: Can you fix the spacing on this line?

Reply:
Sorry my bad , Done now :)